wordpress-mobile/WordPress-iOS
# Issue/7083 slug not pre filled for drafts

**Fixes** #7083 
This is an improvement on: https://github.com/wordpress-mobile/WordPress-iOS/pull/7034
When a slug has not been entered by the user we prefill the slug field with the `suggested_slug` (which will become the `slug` server side if the user doesn't update it)

**To test:**
1. Create a new Post.
2. Save it as a Draft.
3. Go to Options and check that the `slug` field is populated.
4. Update the `slug`
5. Check that that worked too.

Needs review: @kurzee
